### ISSUE TITLE
[BUG] D100 only rolls the 10s dice with 3D on

### ISSUE THREAD

**Issue by ankorite** [Kruptein/PlanarAlly#1529]
**Describe the bug**
When rolling "1d100" with the new dice system and 3D on it only rolls the 10s dice and always reports its results as final.

**To Reproduce**
1. Go to dice roller
2. Click on d100
3. Click on 3D dice ON
4. Click roll

**Expected behavior**
I expect 2d10 to roll with one as the tens and one as the ones. 3D dice off gives expected results

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser chrome
 - Version 131.0.6778.109 (Official Build) (64-bit)

**Additional context**

**Comment by rexy712** [comment]
I've also reported this issue to the planarally-dice repo in https://github.com/Kruptein/planarally-dice/issues/11

My suggestion was to make a separate d% which can be used in both 2D and 3D. Then when selecting the d100 option in 3D, simulate a d10 + d% to get the result.

**Comment by develroo** [comment]
> Then when selecting the d100 option in 3D, simulate a d10 + d% to get the result.

FWIW that is kinda how it works now. Maybe they label should be changed from d100 but percentiles are d100+d10. Like below.


https://github.com/user-attachments/assets/f8b73344-7e4f-4f35-a68d-9585be291afd

**Comment by rexy712** [comment]
True, selecting both d10+d100 would work. But the problem is that the d100 covers the range [10, 100] (increments of 10) and the d10 covers [1, 10]. So when combined the results range from [11, 110], when the desired range is [1, 100].

There's also the discrepancy between 2D and 3D setting. When in 2D, the d100 covers all integers in the range [1, 100]. The 3D version covers increments of 10, as stated above.

**Comment by Kruptein** [comment]
> when the desired range is [1, 100]

The desired range depends on the game system and variant rules. There are also systems that use [0-99].

I'm aware of the issue, it will be resolved.

**Comment by rexy712** [comment]
> The desired range depends on the game system and variant rules. There are also systems that use [0-99].

Good point, hadn't thought of that. I was just stating the desired as [1-100] to align it with what the 2D system currently does

**Comment by develroo** [comment]
> > The desired range depends on the game system and variant rules. There are also systems that use [0-99].
> 
> Good point, hadn't thought of that. I was just stating the desired as [1, 100] to align it with what the 2D system currently does

Well, we went deep into idea [here](https://github.com/Kruptein/PlanarAlly/issues/966) ;)

**Comment by Kruptein** [comment]
This should now be resolved with #1534

**Comment by develroo** [comment]
Indeed, it does. :) And 100 is treated correctly. 

![image](https://github.com/user-attachments/assets/0ea7b633-d9fa-4449-9bbe-ebb04a26fe58)


Cheers!
